File: 02. C# Part2/03. Methods-Homework/04. AppearanceCount/AppearanceCount.cs
using System;
using System.Linq;

        //Write a method that counts how many times given number appears in given array.
        //Write a test program to check if the method is workings correctly.

public class AppearanceCount
{
    static void Main()
    {
        Console.WriteLine("Enter the integers of the array: ");
        int[] numbers = Console.ReadLine()
            .Split(new char[] { ' ', ',', '\t' }, StringSplitOptions.RemoveEmptyEntries)
            .Select(x => int.Parse(x))
            .ToArray();
        Console.WriteLine("Enter a number you want to search: ");
        int searchNumber = int.Parse(Console.ReadLine());
        int index = 0;
        int count = CountElementAppearance(numbers, searchNumber, index);
        Console.WriteLine("The number {0} is found {1} times", searchNumber, count);


    }

    public static int CountElementAppearance(int[] numbers, int searchNumber, int index)
    {
        for (int i = 0; i < numbers.Length; i++)
        {
            if (searchNumber == numbers[i])
            {
                index++;
            }
        }
        return index;
    }
}

File: 06. JavaScript Basics/06. Functions-Homework/06. Functions-Homework/05. AppearanceCount/AppearanceCount.js
//Write a function that counts how many times given number appears in given array.
//Write a test function to check if the function is working correctly.

function appearance(arr, number) {
    var times = 0;
    for (var i = 0; i < arr.length; i++) {
        if (number === arr[i]) {
            times++;
        }
    }
    jsConsole.writeLine("Тhe number " + number + " appears " + times + " times in the array [" + arr + "]");
    
}

appearance([3, 4, 5, 6, 8, 2, 6, 8], 8);